lib/txdb/response_helpers.rb

```diff
@@ -0,0 +1,26 @@
+module Txdb
+  module ResponseHelpers
+    private
+
+    def respond_with(status, body, e = nil)
+      Txdb::Response.new(status, body, e)
+    end
+
+    def respond_with_error(status, message, e = nil)
+      respond_with(status, error(message), e)
+    end
+
+    def error(message)
+      [{ error: message }]
+    end
+
+    def data(body)
+      { data: body }
+    end
```

Comment:
Trying to follow JSON API here. I think errors are supposed to be arrays, although I'm not sure about data. It looks like this method isn't even used.